jupyter_server/__init__.py

```diff
@@ -0,0 +1,13 @@
+"""The Jupyter Server"""
+
+import os
+
+DEFAULT_STATIC_FILES_PATH = os.path.join(os.path.dirname(__file__), "static")
+DEFAULT_TEMPLATE_PATH_LIST = [
+    os.path.dirname(__file__),
+    os.path.join(os.path.dirname(__file__), 'templates'),
+]
```

Comment:
I think that most of the templates besides maybe `404` should be removed. Thoughts?

Reply:
Agreed